_posts/blog/2019-08-13-Ang-Li-Three-People-in-Turing.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Three amazing people in Turing"
+categories: blog
+date: 2019-08-13
+modified:
+share: true
+author: Ang Li
+---
+
+Some amazing people I met in Turing. :)
+
+## Bo Wang
+
+Bo Wang is a visiting researcher at Turing and work with Professor Terry Lyons based at Oxford. 
+I knew him because he is one of the organizers for Data science for mental health, an interest group aiming
+to lead to suitable clinical interventions for mental health by using multi-modal data. Bo is interested in 
+developing clinically useful application for mental health by his expert in deep learning, also he’s working 
```

Comment:
I recommend you change the following: 
application - applications
expert - expertise
also he's working - new sentence. He's also working

Reply:
```suggestion
developing a clinically useful application for mental health through his expert in deep learning. He’s also working 
```